core/src/main/java/org/apache/brooklyn/core/entity/EntityFunctions.java

```diff
@@ -154,7 +154,23 @@ protected GetEntityAttributeFunction(AttributeSensor<T> attribute) {
         @Override public T apply(Entity input) {
             return (input == null) ? null : input.getAttribute(attribute);
         }
-    };
+    }
+
+    public static <T> Function<Object, T> attribute(Entity entity, AttributeSensor<T> attribute) {
+        return new GetFixedEntityAttributeFunction<>(entity, attribute);
```

Comment:
Personal preference for `checkNotNull(entity, "entity"), checkNotNull(attribute, "attribute")`.